java-manta-client-unshaded/src/main/java/com/joyent/manta/client/MantaObjectResponse.java

```diff
@@ -168,6 +178,10 @@ public MantaObjectResponse(final String path, final MantaHttpHeaders headers,
                 this.type = null;
             } else if (contentType.equals(DIRECTORY_RESPONSE_CONTENT_TYPE)) {
                 this.type = "directory";
+            } else if (contentType.equals(BUCKET_RESPONSE_CONTENT_TYPE)) {
+                this.type = "bucket";
```

Comment:
Can/Should this be using the constants from `MantaObject.java` instead of writing out the same strings again?